MarvelApiClient/src/main/java/com/karumi/MarvelApiClient/ApiCall.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *   Copyright (C) 2015 Karumi.
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.karumi.marvelapiclient;
+
+import retrofit.Retrofit;
+
+class ApiCall {
+  private final Retrofit retrofit;
+
+  public ApiCall(Retrofit retrofit) {
+    this.retrofit = retrofit;
+  }
+
+  public <T> T obtainApi(Class<T> apiRest) {
+    return retrofit.create(apiRest);
```

Comment:
We should rename this to use the word ``client``. We are not going to get an instance of a rest api, we are going to get an instance of an api client. My suggestion is:

```java
public <T> T obtainApiClient(Class<T> apiClient) {
     return retrofit.create(apiClient);
}
```

If you want to add ``rest`` to indicate the api is a rest api, ok, but remember that retrofit can be also used to consume a non rest service.